### PR TITLE
Sync 1.14.x releases into changelogs

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.14
* [ ] 1.13
* [ ] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: debian/jessie/foreman-installer/changelog [hunk 1]
```diff
@@ -1,3 +1,39 @@
+foreman-installer (1.14.2-1) stable; urgency=low
+
+  * 1.14.2 released
+
+ -- Daniel Lobato Garcia <me@daniellobato.me>  Thu, 30 Mar 2017 10:15:41 +0200
```
Comment (domcleal) [inline debian/jessie/foreman-installer/changelog]: Please re-check against deb/1.14, I'm pretty sure these have different dates and authors!

File: debian/jessie/foreman-installer/changelog [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+
+ -- Daniel Lobato Garcia <me@daniellobato.me>  Thu, 30 Mar 2017 10:15:03 +0200
+
+foreman-installer (1.14.0~rc1-1) stable; urgency=low
+
+  * 1.14.0-RC1 released
+
+ -- Daniel Lobato Garcia <me@daniellobato.me>  Thu, 30 Mar 2017 10:14:58 +0200
+
 foreman-installer (1.15.0-1) stable; urgency=low
```
Comment (domcleal) [inline debian/jessie/foreman-installer/changelog]: Should be removed.

**Comment by dLobatog** [comment]
Updated according to deb/1.14

**Comment by dLobatog** [comment]
@mmoll or @domcleal Can you cherry pick this to deb/1.15 when it's merged?

**Comment by dLobatog** [comment]
No need for this PR anymore, @mmoll put https://github.com/theforeman/foreman-packaging/commit/602c743c9696a5d5f168c3763bf8da3ae99f006a directly on 1.15.

**Pull request closed without merging**